packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -90,15 +91,26 @@ export function GridFilterForm(props: GridFilterFormProps) {
       currentColumn.filterOperators?.find((operator) => operator.value === item.operatorValue) ||
       null
     );
-  });
+  }, [item, getCurrentColumn]);
+
+  const CurrentOperatorInputComponent = React.useCallback(() => {
```

Comment:
Since it creates a different component each time a dependency is changed, the focus is lost after the debounce. I would revert this part.

Reply:
I was playing with this idea yesterday. The reason why it didn't lose focus before is that the `currentOperator` was kept in the state. and now every time the value changes the `getCurrentOperator()` is called again which results in input loosing focus.